pkg/apis/bus/v1alpha1/actions.go

```diff
@@ -28,10 +28,16 @@ const (
 	// RestartJobAction if this action is set, the whole job will be restarted
 	RestartJobAction Action = "RestartJob"
 
-	// RestartTaskAction if this action is set, only the task will be restarted; default action.
+	// RestartTaskAction if this action is set, only the task will be restarted
+	// It means that all pods under the task will be deleted and recreated.
 	// This action can not work together with job level events, e.g. JobUnschedulable
 	RestartTaskAction Action = "RestartTask"
 
+	// RestartPodAction if this action is set, only the pod will be restarted
+	// It means that only the pod corresponding to the event will be deleted and recreated.
+	// This action can just work together with pod level events, e.g. PodFailed
+	RestartPodAction Action = "RestartPod"
```

Comment:
This change seems to have no relation with network topology.

Reply:
These commits is synced from master branch, will submit a seperate pr later.